### DISCUSSION TITLE
The Gamified Political Compass | Bad Optics: The Gamified Political Compass! - You know what they say, opposites repel.

### DISCUSSION THREAD

**Discussion by giscus** [CanPixel/BadOptics#91] [General]
# The Gamified Political Compass | Bad Optics: The Gamified Political Compass! - You know what they say, opposites repel.

Bad Optics: The Gamified Political Compass! - You know what they say, opposites repel.

https://canpixel.github.io/BadOptics/

**Comment by CanPixel** [comment]
Governing a country is hard, there is no one way.
Go figure it out!

**Reply by RickStikkelorum** [reply]
Incorrect, there is only communism

**Comment by FlowmaticGH** [comment]
I'm pretty sure there is a bug where some questions give you the opposite result. Like I clicked "Wars should be avoided at all costs" and I hit disagree and I got futurist and pacifist.

**Reply by CanPixel** [reply]
Yes, sometimes you get inconsistencies in the citizen responses. This is working correctly tho, and it's a pretty technical point to explain. They don't specifically look at individual questions and answers, that would've been very tedious and gigantic to implement. So to say: They specifically look at the effects it has on you as a state ideology, after answering the questions. I consciously made it this way, so it has more dynamics to it and room for expansion in the future.

**Reply by navavava** [reply]
hi! i also have this problem in which i answer a question and the game changes my status accordingly but the result i get from my population is usually( like 80% of the time) the opposite of what i expect them to be. for example i answered "agree" to the q "means of production should be distributed among workers" or something along the lines of that and my "aristocracy" population absolutely loved it while another party with the liberty and equality as its prominent features had no reaction

**Comment by pastel-pink-python** [comment]
If there isn't Eco-Posadism I'm quitting

**Reply by CanPixel** [reply]
Adding it in the next patch!

**Comment by dietxlo** [comment]
Rather than using block + inline-block display + zooming in/out when things overlap , I recommend using flex or grid css.

**Comment by batnode07** [comment]
decided that i was going to become INGSOC immediately

**Comment by ghost** [comment]
Ok, who tf is adding like 60 "Add sex"'s to the google survery?? tf wrong with u bro

**Comment by warleo474** [comment]
This is the third time that while playing Bad Optics my progress has been deleted, as in all my collected ideologies are gone. This isn't after I close out the game or finish playing, it is DURING gameplay that it disappears. I spent hours playing and am really confused and frustrated.

**Comment by CrtyxScuizx** [comment]
Can you make Bad Optics for mobile?

**Comment by greenjustin2008** [comment]
good game

**Comment by 2x2Master** [comment]
Add eshayism or you will be banned from cinemas the second shrek 5 comes out

**Comment by Wolfe-Flameheart** [comment]
Can you please add Dreemurrism into the game?

**Comment by RusMermaid** [comment]
I want Anarcho-Nazbol, Communo-Capitalism, Blue Socialism and most importantly True Post-Singularity
